client/settings/components/shipping-services/group.js

```diff
@@ -56,7 +49,7 @@ const ShippingServiceGroup = ( props ) => {
 	const onChange = ( event ) => updateAll( event, updateValue, services );
 
 	const renderHeader = () => {
-		return <div className="wcc-shipping-services-group-header">
+		return <div className="shipping-services__group-header">
```

Comment:
Looks like we don't :)
Removed in https://github.com/Automattic/woocommerce-services/pull/1051/commits/22d672f42b114682bf1c1ff382c0859fccd27cd4